### PR TITLE
minor README additions: 1) note about fixing libffi build issue on Ma…

### PR DESCRIPTION
**Pull request by elenzil**
…cOS 2) Notes about how to build & run grimoire, for the rust-noobs in the audience.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +86,27 @@ PKG_CONFIG_PATH="/c/Users/jshrake/scoop/apps/msys2/current/mingw64/lib/pkgconfig
 Breadcrumbs:
 - https://github.com/sdroege/GStreamer-rs#windows
 
+### Build
+Rust's build system is "cask".
```
Comment (jshrake) [inline README.md]: Replace "cask" with "cargo".

File: README.md [hunk 2]
```diff
@@ -80,6 +86,27 @@ PKG_CONFIG_PATH="/c/Users/jshrake/scoop/apps/msys2/current/mingw64/lib/pkgconfig
...
+
+```console
+$ cask build
```
Comment (jshrake) [inline README.md]: Replace "cask" with "cargo".

File: README.md (hunk 1 above)
Comment (elenzil) [reply README.md]: ah. yes.

**Comment by elenzil** [comment]
fixed.

**Comment by jshrake** [comment]
Awesome, thanks again!

**Pull request merged**